app/services/clients/stripe/stripeCompany.model.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+// NPM dependencies
+const Joi = require('joi')
+
+const schema = {
+  vat_id: Joi.string().required(),
+  tax_id: Joi.string().optional()
```

Comment:
Isn’t `.optional()` the default?

Reply:
yes, it is, I wanted to be explicitly clear that it is optional as per docs https://github.com/hapijs/joi/blob/v14.3.1/API.md#anyoptional
```
Used to annotate the schema for readability as all keys are optional by default.
```

Reply:
Fair enough — do you think we should also do this in `stripePerson.model.js`?

Reply:
If we all agree, probably yes, for consistency. I was thinking to make it explicit as in Java `Optional`